### PR TITLE
Use standard port for PostgreSQL.

### PR DESCRIPTION
**Pull request by AlekSi**
There is no port clash with pmm-managed anymore because it uses devcontainer.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -57,7 +57,7 @@ services:
       -c pg_stat_statements.track=all
       -c pg_stat_statements.save=off
     ports:
-      - 127.0.0.1:15432:5432
+      - 127.0.0.1:5432:5432
```
Comment (BupycHuk) [inline docker-compose.yml:60]: it was done because it was conflicting on my local system

Comment (BupycHuk) [reply docker-compose.yml:60]: with local installed postgresql

Comment (AlekSi) [reply docker-compose.yml:60]: But do you use it?

My motivation was to make it behave the same as MySQL and MongoDB for local checks development:
```
pmm-admin add mysql
pmm-admin add mongodb
pmm-admin add postgresql postgresql 127.0.0.1:15432
```

Comment (AlekSi) [reply docker-compose.yml:60]: @BupycHuk ^

**Pull request merged**